static/src/javascripts-legacy/projects/common/modules/navigation/search.js

```diff
@@ -63,7 +68,6 @@ define([
                 // Make sure search is always in the correct state
                 self.focusSearchField();
                 e.preventDefault();
-                mediator.emit('modules:search');
```

Comment:
You are totally right <3. I decided too late, I'd need a refactoring PR like this: https://github.com/guardian/frontend/pull/16831 ;)